ngx_http_vod_module.c

```diff
@@ -64,6 +64,8 @@ typedef ngx_int_t(*ngx_http_vod_open_file_t)(ngx_http_request_t* r, ngx_str_t* p
 typedef ngx_int_t(*ngx_http_vod_async_read_func_t)(void* context, ngx_buf_t *buf, size_t size, off_t offset);
 typedef ngx_int_t(*ngx_http_vod_dump_part_t)(void* context, off_t start, off_t end);
 typedef size_t(*ngx_http_vod_get_size_t)(void* context);
+typedef ngx_int_t (*ngx_http_vod_enable_directio_t)(ngx_file_reader_state_t* state);
```

Comment:
please change the parameter to `void* context`, this is intended as an abstraction, so should not be tied to the file reader (even though that's the only implementation)